coreos-base/afterburn/files/0001-Revert-remove-cl-legacy-feature.patch

```diff
@@ -0,0 +1,591 @@
+From ba861f48c0ca6abb0944c362b4bf9e79bfbb7bc8 Mon Sep 17 00:00:00 2001
```

Comment:
Looks ok but I doubt this large patch will cleanly apply after a few months, thus the idea of using `sed` in `ExecStartPost`https://github.com/flatcar-linux/Flatcar/issues/387#issuecomment-1080532259

Reply:
Yes - I just find more consistent to get every changes applied in one place. Something we can do: reduce this patch to bring back only  `src/providers/vagrant_virtualbox/mod.rs` and `sed` the `AWS/GCP/AFTERBURN` prefixes.